docs/guide/your-first-scene.md

```diff
@@ -10,13 +10,13 @@ This guide will help you to create your first Tres scene. 🍩
 
 ## Setting up the experience Canvas
 
-Before we can create a Scene, we need somewhere to display it. Using plain [ThreeJS](https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene) we would need to create a `canvas` HTML element to mount the `WebglRenderer` and initialize the `scene`
+Before we can create a scene, we need a space to display it. Using plain [ThreeJS](https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene) we would need to create a `canvas` HTML element to mount the `WebglRenderer` and initialize the `scene`
```

Comment:
(non-blocking)

> Before we can create a ThreeJS `Scene`, we need somewhere to display it. Using plain [ThreeJS](https://threejs.org/docs/index.html#manual/en/introduction/Creating-a-scene) we would need to create a `<canvas>` HTML element to mount the `WebglRenderer` and initialize the `Scene`.

----

## Explanation

* To me, it's clearer when HTML looks like tags (`<canvas>`).
* Made "Scene" and `scene` consistent. It's a JS class, so typically capitalized so `Scene`.
* Added a missing period at the end.